IntegrationDevelopment/Help/IntegrationDevelopmentGuide.postman_collection.json

```diff
@@ -229,6 +229,115 @@
 						}
 					]
 				},
+				{
+					"name": "New journal transaction with changed business date and branch",
```

Comment:
@PetrKorchagin , please change the name to the name of the topic in help.
Do not forget to change the name in the example of response.

Reply:
Done.